recipes/fusesoc/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "fusesoc" %}
+{% set version = "1.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 47589cc47583a9d4f0a6f57bd40555f1518ab2317c2526041f3bc0e366317874
+  patches:
+    - remove_jsonschema_dependency.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - edalize >=0.1.3
+    - ipyxact >=0.2.3
+    - pyparsing
+    - pytest >=3.3.0
+    - pyyaml
+    - simplesat >=0.8.0
+
+test:
+  imports:
+    - fusesoc
+
+about:
+  home: https://github.com/olofk/fusesoc/blob/master/LICENSE
+  license: GPLv3
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-3.0
```